### DISCUSSION TITLE
Custom import type for a table!

### DISCUSSION THREAD

**Discussion by oldner** [diesel-rs/diesel#4052] [Q&A] [answered]
Hi, is there a way to add `import_type` to only for a specific `table!` and not all of them in the `diesel.toml` file? The thing is I have more than 150 tables and only 1 will use `diesel_full_text_search::TsVector` but when I define it in the `import_types`, it adds this to all `table!`s. Any help will be appreciated.

**Comment by weiznich** [comment, marked as answer]
The `import_type` config field is always applied to all of your tables as we don't know for which it is relevant in the general case. We do currently not offer any feature that would allow you to specifically define an import only for a single table. You could likely use the patch-file support to achieve something like that by generating the corresponding patch file on your own.
